gym/utils/play.py

```diff
@@ -18,6 +21,55 @@
 from pygame.locals import VIDEORESIZE
 
 
+class PlayableGame:
+    def __init__(self, env: Env, keys_to_action: dict = None, zoom: float = None):
```

Comment:
`keys_to_action: Optional[dict]`

Reply:
Even better if we can specify the key/value types on the dict (I guess values are arbitrary actions, but keys should be keypresses? how are they represented?)

Reply:
It's ```Dict[Tuple[int], int]``` where the key is ```(ord(key),)```
https://github.com/openai/gym/blob/5ae6bf9634c73ed09d73ceeb41e3491d7e791cce/gym/envs/classic_control/mountain_car.py#L240-L242  

reference of the actions:
https://github.com/openai/gym/blob/5ae6bf9634c73ed09d73ceeb41e3491d7e791cce/gym/envs/classic_control/mountain_car.py#L50-L54

https://github.com/openai/gym/blob/9a1d4b67684785c03c187c0a7e8ab217b8f9a28f/gym/utils/play.py#L125-L135